storage/container_storage_interface/persistent-storage-csi-ebs.adoc

```diff
@@ -1,51 +1,51 @@
+* The _Amazon EBS CSI Driver Operator_ provides a StorageClass by default that you can use to create PVCs. You can disable this default storage class if desired (see xref:../../storage/container_storage_interface/persistent-storage-csi-sc-manage.adoc#persistent-storage-csi-sc-manage[Managing the default storage class]). You also have the option to create the Amazon EBS StorageClass as described in xref:../../storage/persistent_storage/persistent-storage-aws.adoc#persistent-storage-aws[Persistent storage using Amazon Elastic Block Store]. 
 
-* The _AWS EBS CSI driver_ enables you to create and mount AWS EBS PVs.
+* The _Amazon EBS CSI driver_ enables you to create and mount Amazon EBS PVs.
 
 ifndef::openshift-rosa,openshift-dedicated[]
 [NOTE]
 ====
-If you installed the AWS EBS CSI Operator and driver on an {product-title} 4.5 cluster, you must uninstall the 4.5 Operator and driver before you update to {product-title} {product-version}.
+If you installed the Amazon EBS CSI Operator and driver on an {product-title} 4.5 cluster, you must uninstall the 4.5 Operator and driver before you update to {product-title} {product-version}.
 ====
 endif::openshift-rosa,openshift-dedicated[]
 
 include::modules/persistent-storage-csi-about.adoc[leveloffset=+1]
 
 [IMPORTANT]
 ====
-{product-title} defaults to using the CSI plugin to provision AWS EBS storage.
+{product-title} defaults to using the CSI plugin to provision Amazon EBS storage.
 ====
 
-For information about dynamically provisioning AWS EBS persistent volumes in {product-title}, see xref:../../storage/persistent_storage/persistent-storage-aws.adoc#persistent-storage-aws[Persistent storage using AWS Elastic Block Store].
+For information about dynamically provisioning Amazon EBS persistent volumes in {product-title}, see xref:../../storage/persistent_storage/persistent-storage-aws.adoc#persistent-storage-aws[Persistent storage using Amazon Elastic Block Store].
 
 ifndef::openshift-rosa,openshift-dedicated[]
 include::modules/persistent-storage-byok.adoc[leveloffset=+1]
 
 [NOTE]
 ====
-If there is no encrypted key defined in the storage class, only set `encrypted: "true"` in the storage class. The AWS EBS CSI driver uses the AWS managed alias/aws/ebs, which is created by Amazon EBS automatically in each region by default to encrypt provisioned storage volumes. In addition, the managed storage classes all have the `encrypted: "true"` setting.
+If there is no encrypted key defined in the storage class, only set `encrypted: "true"` in the storage class. The Amazon EBS CSI driver uses the AWS managed alias/aws/ebs, which is created by Amazon EBS automatically in each region by default to encrypt provisioned storage volumes. In addition, the managed storage classes all have the `encrypted: "true"` setting.
```

Comment:
Revoked.

Reply:
https://github.com/openshift/aws-ebs-csi-driver-operator  is a Red Hat operator with just that name. If we mention AWS EBS CSI driver (not the Red Hat operator, but the driver itself), then it does make sense to expand it because the source for the Red Hat fork of the driver is https://github.com/kubernetes-sigs/aws-ebs-csi-driver, which in the description has the expanded name: CSI driver for Amazon EBS.

Reply:
Updated, "If there is no encrypted key defined in the storage class, only set encrypted: "true" in the storage class. The AWS EBS CSI driver uses the AWS managed alias/aws/ebs, which is created by Amazon Elastic Block Store (EBS) automatically in each region by default to encrypt provisioned storage volumes. In addition, the managed storage classes all have the encrypted: "true" setting."